### PR TITLE
Contribute page

### PR DESCRIPTION
**Pull request by MichaIng**
Just to get an overview of style changes.

### CODE REVIEW

File: contribute.html [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Investigate issues / Provide help</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p>Many times the DietPi project needs help on existent items:
+								<ul>
+									<li>Run an investigation (e.g. an installation issue on specific software title)</li>
+									<li>Provide an workaround to an existent issue</li>
```
Comment (StephanStS) [inline contribute.html:132]: "an workaround" -> "a workaround"

File: contribute.html [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+					<div class="col-md-4">
+						<a href="images/contribute/question.jpg" target="_blank" rel="noopener">
+							<img src="images/contribute/question.jpg" alt="Question" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Give help / Answer questions</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> There are many questions on GitHub, and also on Forum. Maybe for some of the issues, we need more insights.</p>
```
Comment (StephanStS) [inline contribute.html:152]: "on Forum." -> "in the Forum."

File: contribute.html [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+				<div id="slidingdiv4" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/test-configurations.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/test-configurations.jpg" alt="GNUstep" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Test configurations</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> Very similar with investigate issues, but more specific to hardware - either specific SBC models or PCs</p>
```
Comment (StephanStS) [inline contribute.html:171]: `...specific SBC models or PCs</p>` -> `...specific SBC models or PCs.</p>`

Full stop at end of sentence is missing.

File: contribute.html [hunk 4]
```diff
@@ -0,0 +1,434 @@
...
+							<p>There is often a need to check a new installation on a specific SBC (Single Board Computer), or maybe how DietPi runs with a specific driver. This time to do such installation provides a big help to the overall project.</p>
+							<p>Examples:
+								<ul>
+									<li>A driver installation for a specific SBC</li>
+									<li>or check of a performance tweak.</li>
```
Comment (StephanStS) [inline contribute.html:176]: "or check" -> "A check".

Otherwise put the "or" one line above at the end.

File: contribute.html [hunk 5]
```diff
@@ -0,0 +1,434 @@
...
+					<div class="col-md-4"><a href="images/contribute/improve-website.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/improve-website.jpg" alt="Chromium" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Improve DietPi website</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> On the website it's always something to do. Someone with skills in graphics / web design could provide help.</p>
+							<p>Have you seen something on the website which could be improved ? Do you have web design skills and you would like to help ?</p>
```
Comment (StephanStS) [inline contribute.html:195]: No blank before question marks. 2x.

e.g. "... improved ?" -> "improved?"

File: contribute.html [hunk 6]
```diff
@@ -0,0 +1,434 @@
...
+				<div id="slidingdiv7" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/new-features.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/new-features.jpg" alt="NoMachine" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>New feature suggestions & Software titles</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> Let's add links to GitHub add issues (similar with what we have now on DietPi, we should also create on DietPi-Docs, and DietPi-Website)</p>
```
Comment (StephanStS) [inline contribute.html:227]: Full stop at end of sentence is missing.

File: contribute.html [hunk 7]
```diff
@@ -0,0 +1,434 @@
...
+				<div id="slidingdiv8" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/expand-code.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/expand-code.jpg" alt="XRDP" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Extend DietPi</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> This section is all about the code. Mainly if they have the expertise to help us fixing / enhancing DietPi project</p>
```
Comment (StephanStS) [inline contribute.html:243]: `enhancing DietPi project</p>` -> `enhancing the DietPi project.</p>`

Added "the" and full stop.

File: contribute.html [hunk 8]
```diff
@@ -0,0 +1,434 @@
...
+							<p>Bring your contributions to DietPi projects.</p>
+							<p>DietPi project.</p>
```
Comment (StephanStS) [inline contribute.html:245]: Delete line above.

File: contribute.html [hunk 9]
```diff
@@ -0,0 +1,434 @@
...
+				<div id="slidingdiv9" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/be-part-community.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/be-part-community.jpg" alt="RealVNC" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Use DietPi Forum</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> Browse, found things to contribute and help</p>
```
Comment (StephanStS) [inline contribute.html:260]: `</b> Browse, found things to contribute and help</p>` -> `</b> Browse found things to contribute and help.</p>`

File: contribute.html [hunk 10]
```diff
@@ -0,0 +1,434 @@
...
+				<div id="slidingdiv10" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/social-Twitter.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/social-Twitter.jpg" alt="Kodi" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Write about DietPi</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> It helps a lot to write about project, experience using DietPi etc. - blog posts, youtube links. Link to reference page</p>
```
Comment (StephanStS) [inline contribute.html:276]: Full stop at end of sentence is missing.

**Review by StephanStS — CHANGES_REQUESTED** [review]

**Comment by fpetru** [comment]
@StephanStS - thanks, I'll review all of the comments

**Review by StephanStS — APPROVED** [review]

**Comment by fpetru** [comment]
@MichaIng - please review and merge this PR.

File: contribute.html [hunk 11]
```diff
@@ -0,0 +1,434 @@
...
+							<img src="images/contribute/diagnose2.jpg" alt="Bug report" loading="lazy">
+						</a>
+					</div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Create a new bug report / Report a security issue</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p>Creating a bug report helps us improve. Providing required info, steps to replicate, and expected vs. actual behaviour you give us valuable info to understand the problem.
```
[the remark this answers is not in the document]
Comment (MichaIng) [reply contribute.html:108]: According to deepl.com, both is correct, but "to improve" is the first offer, hence probably more usual.

File: contribute.html (hunk 4 above)
Comment (MichaIng) [reply contribute.html:181]: Or "Testing a performance tweak"?

File: contribute.html (hunk 5 above)
Comment (MichaIng) [reply contribute.html:200]: Probably a style/visual question as well, but let's stick with correct notation and skip the space between words and punctuation in all cases.

**Comment by MichaIng** [comment]
Since SEO scans will complain: Shall we degrade the second and third header to `<h2>`?

**Comment by MichaIng** [comment]
New build: https://dietpi.com/test/contribute.html
LGTM! 😃

**Review by fpetru — APPROVED** [review]

**Pull request merged**